### PR TITLE
feat: stacking part 1 show no balance

### PR DESCRIPTION
**Pull request by rodrigopavezi**
@sk91 this is still a working in progress as we cannot merge to dev. We need the full stacking for that. But it is good to separate in small chunks for make it easier for review. Cheers

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -117,9 +117,10 @@
     "request": "2.87.0",
     "reselect": "^3.0.1",
     "scrypt": "^6.0.3",
-    "selfkey-ui": "https://github.com/SelfKeyFoundation/selfkey-ui.git",
+    "selfkey-ui": "https://github.com/SelfKeyFoundation/selfkey-ui.git#feat/stack-1",
```
Comment (sk91) [inline package.json]: using hashes instead of branches works great after merge, better to use them

File: src/common/exchanges/reducers.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	list: []
+};
+
+const exchangesReducer = (state = initialState, action) => {
+	switch (action.type) {
+		case types.EXCHANGES_UPDATE:
+			console.log('Exchanges', action);
+			return {
+				...state,
+				list: action.payload
```
Comment (sk91) [inline src/common/exchanges/reducers.js]: list should store ids/keys only with byId near it to normalize state

File: package.json (hunk 1 above)
Comment (rodrigopavezi) [reply package.json]: I know this is just for now that is why it is merging to staging/stacking. It will be changed when merging to dev

**Comment by sk91** [comment]
Need to do something with the coverage though ...

**Comment by rodrigopavezi** [comment]
@sk91 yeap, I will add tests for the redux stuff so our coverage continues to increase.

**Pull request merged**